test/box/gh-6405-discarding-arguments-in-box-func-call.test.lua

```diff
@@ -0,0 +1,29 @@
+args_type = function(...)                     \
+    local type_table = {}                     \
+    for i = 1, select('#', ...) do            \
+        type_table[i] = type(select(i, ...))  \
+    end                                       \
+    return type_table                         \
+end                                           \
+
+box.schema.func.create('args_type')
```

Comment:
I guess you should drop this func at the end of the test.

Reply:
Fixed